social-network/src/YourArticle.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+// import PropTypes from 'prop-types';
+
+class YourArticle extends Component {
+    render () {
+        const article = this.props.post
```

Comment:
Since this component uses a lot of attributes within article, you'll want some proptypes.